index.js

```diff
@@ -108,6 +108,13 @@ exports.decorateConfig = (config) => {
DggOC4xMyAxMS45NTUgMTAuOCAyLjA4NCAxLjIxNCAzLjY4 MyAwLjcyNiA1LjI2NyAwLjUxMiAwLjUwOSAwLjM3NiAyLjIyNCAxLjYwNSAzLjIw NCAyLjI3NCAxNS4wOTMtMTYuMDE0IDM0LjkxNy0yNy44NzQgNTcuOTQyLTMzLjEw MyA1LjM1LTEuMjEzIDEwLjcwOS0yLjAyMiAxNi4wMjgtMi40NjNsMC4yMTEtMy43 NTFjMS4xODEtMS4xNDMgMi41MDQtMi43ODYgMi44ODEtNC41ODYgMC4zODQtNC43 NjctMC4yNC05Ljg5Ny0wLjk4Ni0xNi4wODUtMC40MTItMi44OTMtMS4wOTYtNS4y OTQtMS4yMTQtOC40MzItMC4wMjMtMC42NDYgMC4wMTItMS41MzcgMC4wMjEtMi4y NzQtM2UtMyAtMC4wODMtMC4wMjEtMC4xNjMtMC4wMjEtMC4yNDkgMC01LjQzOSAz Ljk3LTkuODUgOC44NjctOS44NTMgNC45IDAgOC44NjggNC40MTQgOC44NzEgOS44 NTMgMmUtMyAwLjc3MiAwLjA0NCAxLjgwOSAwLjAxNyAyLjUyMi0wLjExNSAzLjEz OC0wLjgwMyA1LjUzOS0xLjIxNCA4LjQzMi0wLjc0NiA2LjE4OC0xLjM4OSAxMS4z MTgtMS4wMDQgMTYuMDg1IDAuMzUzIDIuMzg3IDEuNzQ3IDMuMzMxIDIuODk4IDQu NDM1IDAuMDI3IDAuNjQzIDAuMTMgMi43NzcgMC4yMTEgMy45NTkgMjguMTk0IDIu NTA0IDU0LjM4MSAxNS4zOTggNzMuNjEzIDM1LjU2NWwzLjM3Mi0yLjQwNmMxLjE0 MyAwLjA2OCAzLjY2MiAwLjQxNSA1LjM4My0wLjU4NyAzLjk2Ny0yLjY3IDcuNTgx LTYuMzc3IDExLjk1NS0xMC44MTggMi4wMDQtMi4xMjYgMy40NzMtNC4xNDggNS44 NTYtNi4xOTcgMC41NDEtMC40NjggMS4zNjQtMS4wOTIgMS45NjgtMS41NzIgNC4y NTQtMy4zOTYgMTAuMTczLTMuMDM1IDEzLjIyNSAwLjc5NCAzLjA1NiAzLjgzMSAy LjA3NiA5LjY4MS0yLjE3OSAxMy4wNzQtMC41OTggMC40ODItMS4zNzkgMS4xNTEt MS45NTEgMS41NzUtMi41MjUgMS44NjUtNC44NTIgMi44MTktNy4zNjggNC4yOTkt NS4zMDUgMy4yNzctOS43MDIgNS45OTItMTMuMTkgOS4yNjYtMS42NDYgMS43NjEt MS41MjIgMy40MzItMS42NjcgNS4wMjEtMC40OTEgMC40NTMtMi4yMjcgMS45OS0z LjE0NCAyLjgyNSA3LjY0MSAxMS4zNzUgMTMuMzczIDI0LjI4OSAxNi42MTQgMzgu NDA1IDMuMjE2IDEzLjk4OSAzLjcxNiAyNy45NjkgMS44NiA0MS40MDFsMy41Nzkg MS4wNDJjMC42NDYgMC45MTIgMS45NjkgMy4xMjYgMy44MjggMy44NDkgNC41NjMg MS40MzUgOS42OTkgMS45NjMgMTUuODk5IDIuNjE0IDIuOTEgMC4yNDMgNS40MjEg MC4wOTggOC41MDUgMC42ODEgMC43NCAwLjEzOSAxLjgxOCAwLjQzNiAyLjU3OSAw LjYwNyA1LjIyNSAxLjI2MyA4LjU4MiA2LjA2OCA3LjUwMiAxMC44MDJ6Ii8+Cgk8 cGF0aCBkPSJtMTYzLjM0LTczLjIyN2wtMzQuNjM2IDMwLjk5OCAwLjAxNyAwLjA5 NWMtMS4yMDggMS4wNS0yLjA5OSAyLjQ3Mi0yLjQ4MSA0LjE1LTAuOTM2IDQuMSAx LjUzMyA4LjE4NiA1LjU1MSA5LjI4NGwwLjAzOSAwLjE5IDQ0Ljg2OCAxMi45MjNj MC45NS05LjY4MSAwLjM3My0xOS42OTctMS45MzMtMjkuNzMxLTIuMzMzLTEwLjE1 NC02LjI1LTE5LjUyMS0xMS40MjUtMjcuOTA5eiIvPgoJPHBhdGggZD0ibTUzLjcz OSA0LjUxNmMtMC44MTctMy41ODUtMy45NzktNi4wMzMtNy41MDItNi4wODEtMC41 MDMtOWUtMyAtMS4wMjIgMC4wMzktMS41MzMgMC4xMzNsLTAuMDc3LTAuMDk1LTQ1 Ljc3OCA3Ljc2OWM3LjA2NCAxOS41NDYgMjAuMjcxIDM1LjQ3NyAzNi44NTQgNDYu MTE3bDE3LjczNC00Mi44Ni0wLjEzMy0wLjE2OWMwLjYzMy0xLjQ3MSAwLjgyLTMu MTM2IDAuNDM1LTQuODE0eiIvPgoJPHBhdGggZD0ibTEyNS41OS0xLjYyMWMtMC41 ODktMC4xMS0xLjE5My0wLjE2Ni0xLjgtMC4xMzMtMS4wMSAwLjA1LTIuMDIyIDAu MzA4LTIuOTkzIDAuNzc1LTMuNzkgMS44MjQtNS40MzYgNi4zMTMtMy43OSAxMC4x MzdsLTAuMDU2IDAuMDc3IDE3LjkyMyA0My4yOTZjMTcuMjc4LTExLjAwNyAzMC4y MTEtMjcuNDU3IDM2Ljk3LTQ2LjQ0M2wtNDYuMTc3LTcuODA0LTAuMDc3IDAuMDk1 eiIvPgoJPHBhdGggZD0ibTM4LjUyNC0yOC40NzFjMS41NDItMC40MjQgMi45ODIt MS4zMjYgNC4wNTYtMi42NzMgMi42Mi0zLjI4NiAyLjE1OC04LjAyOS0wLjk4Ni0x MC43NjJsMC4wMzktMC4xODktMzQuODI1LTMxLjE1Yy0xMC4zMzkgMTYuODM3LTE1 LjIzMyAzNy4wMzQtMTIuOTYyIDU3LjgwOWw0NC42NC0xMi44ODUgMC4wMzgtMC4x NXoiLz4KPC9nPgo8L3N2Zz4K');
```

Comment:
Can you switch this to [this SVG](https://github.com/kubernetes/website/blob/master/content/en/docs/tutorials/kubernetes-basics/public/images/logo.svg)? The current icon has a thin white line inside it.